source/acquaman/REIXS/REIXSXESScanActionController.h

```diff
@@ -23,7 +23,6 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
 #define REIXSXESSCANACTIONCONTROLLER_H
 
 #include "acquaman/AMScanActionController.h"
-#include "acquaman/SGM/SGMFastScanConfiguration2013.h"
```

Comment:
Is this file still used in REIXS? I imagine we removed this when we did the initial setup of the project, and moved out all the old files. Was this here for a reason, and do we need to do something about it?